src/ResourceManager/Compute/Commands.Compute/Models/UploadParameters.cs

```diff
@@ -39,7 +39,7 @@ public UploadParameters(BlobUri destinationUri, BlobUri baseImageUri, FileInfo l
 
         public int NumberOfUploaderThreads { get; private set; }
 
-        public AddAzureVhdCommand Cmdlet { get; set; }
+        public ComputeClientBaseCmdlet Cmdlet { get; set; }
```

Comment:
Again, I don't think this is necessary if we model this cmdlet correctly

Reply:
Would need to address this part in a separate PR.